packages/block-editor/src/components/link-control/search-input.js

```diff
@@ -55,34 +56,52 @@ const LinkControlSearchInput = ( {
 
 	return (
 		<form onSubmit={ selectSuggestionOrCurrentInputValue }>
-			<URLInput
-				className="block-editor-link-control__search-input"
-				value={ value }
-				onChange={ selectItemHandler }
-				onFocus={ selectItemHandler }
-				onKeyDown={ ( event ) => {
-					if ( event.keyCode === ENTER ) {
-						return;
+			<div className="block-editor-link-control__search-input-wrapper">
+				<URLInput
+					className="block-editor-link-control__search-input"
+					value={ value }
+					onChange={ selectItemHandler }
+					onFocus={ selectItemHandler }
+					placeholder={ __( 'Search or type url' ) }
+					__experimentalRenderSuggestions={ renderSuggestions }
+					__experimentalFetchLinkSuggestions={ fetchSuggestions }
+					__experimentalHandleURLSuggestions={ true }
+					__experimentalShowInitialSuggestions={
+						showInitialSuggestions
 					}
-					handleLinkControlOnKeyDown( event );
-				} }
-				onKeyPress={ handleLinkControlOnKeyPress }
-				placeholder={ __( 'Search or type url' ) }
-				__experimentalRenderSuggestions={ renderSuggestions }
-				__experimentalFetchLinkSuggestions={ fetchSuggestions }
-				__experimentalHandleURLSuggestions={ true }
-				__experimentalShowInitialSuggestions={ showInitialSuggestions }
-			/>
-			<div className="block-editor-link-control__search-actions">
-				<Button
-					type="submit"
-					label={ __( 'Submit' ) }
-					icon="editor-break"
-					className="block-editor-link-control__search-submit"
 				/>
+				<div className="block-editor-link-control__search-actions">
+					<Button
+						type="submit"
+						label={ __( 'Submit' ) }
+						icon="editor-break"
+						className="block-editor-link-control__search-submit"
+					/>
+				</div>
+				<div role="alert" aria-live="assertive">
+					{ errorMessage && (
+						<Notice
+							className="block-editor-link-control__search-error"
+							status="error"
+							isDismissible={ false }
+						>
+							{ errorMessage }
+						</Notice>
+					) }
+				</div>
 			</div>
+
+			{ errorMessage && (
+				<Notice
+					className="block-editor-link-control__search-error"
+					status="error"
+					isDismissible={ false }
+				>
+					{ errorMessage }
+				</Notice>
+			) }
```

Comment:
Probably caused by rebase, but I see two `Notice` here, when I would expect there to be at most one.

Reply:
Oh this is embarrassing. Will fix.

Reply:
Fixed.